dataframes/01-data-access.ipynb

```diff
@@ -11,27 +11,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    client = Client(n_workers=1, threads_per_worker=4, processes=True, memory_limit='2GB')
</pre><p>This change is needed due to https://github.com/dask/dask/issues/8581</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dask/dask-examples/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>